README.md

```diff
@@ -1,31 +1,36 @@
 # Life Cycle Models
 
 [![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
-[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/OpenSourceEconomics/lcm/main.svg)](https://results.pre-commit.ci/latest/github/OpenSourceEconomics/lcm/main)
-[![image](https://codecov.io/gh/OpenSourceEconomics/lcm/branch/main/graph/badge.svg)](https://codecov.io/gh/OpenSourceEconomics/lcm)
+[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/OpenSourceEconomics/pylcm/main.svg)](https://results.pre-commit.ci/latest/github/OpenSourceEconomics/pylcm/main)
```

Comment:
```suggestion
[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/OpenSourceEconomics/PyLCM/main.svg)](https://results.pre-commit.ci/latest/github/OpenSourceEconomics/PyLCM/main)
```
Doesn't matter, ofc, but if we CamelCase OpenSourceEconomics, looks better if we do the same for PyLCM. Same below.

Reply:
Since we had mixed-use cases of lowercase and CamelCase for OpenSourceEconomics itself, I have decided to just do everything in lowercase. Seems to be more in line with, e.g., PyTorch.